docs/features/where.md

```diff
@@ -7,4 +7,10 @@ parent: Base Features
 grand_parent: Features
 ---
 
```

Comment:
Hey @ilyanaDev 

You're doing a great job here, thanks for the help.
We might add the following sentence at the beginning, and then you can go and describe it in more detail. Folks using Linq are already quite familiar with this operation, so it will be clear for them.

> The `Where` feature defined in the specification behaves the same as `Where` in `Linq`, and it accepts `Expression<Func<TSource, bool>>` expression as a parameter.

We can start in the same manner for the other features too.

Reply:
sounds good